app/src/userpages/modules/userPageStreams/actions.js

```diff
@@ -260,10 +260,6 @@ export const getStream = (id: StreamId) => (dispatch: Function) => {
         .then((id) => dispatch(getStreamSuccess(id)))
         .catch((e) => {
             dispatch(getStreamFailure(e))
-            Notification.push({
```

Comment:
This and the other action is \*all\* about being an action. No side effects like triggering notifications here.

Reply:
yeah notifications should have never been in here